src/NuGet.Core/NuGet.Packaging/Signing/Native/NativeMethods.cs

```diff
@@ -390,4 +390,4 @@ internal struct CERT_CONTEXT
         public IntPtr pCertInfo;
```

Comment:
Undo changes to this file.

Reply:
Since `insert_final_newline = true` in NuGet.Client/.editorconfig. It adds a new line by default when I save it.

Reply:
Exclude this file from your PR.